src/main/java/org/zalando/sprocwrapper/sharding/ShardedDataAccessException.java

```diff
@@ -13,6 +13,8 @@
  */
 public class ShardedDataAccessException extends DataAccessException {
 
+    private static final long serialVersionUID = 1L;
```

Comment:
Why we need it this? Is it serialized?

Reply:
vscode warning, just wanted it all green.